### DISCUSSION TITLE
How to fix "Consider checking `mounted`." error?

### DISCUSSION THREAD

**Discussion by mycar98765** [rrousselGit/riverpod#2362] [Q&A]
I've read this [issue](https://github.com/rrousselGit/riverpod/issues/496), but I've tried all the solutions provided by users and still can't fix this error.

Here is my code:

```dart
await controller.submit(
  …
);
``` 

```dart
Future<void> createBroadcast(…) async {
 state = state.copyWith(value: const AsyncValue.loading());
  final value = await AsyncValue.guard(() async {
    …

    return await …);
  });
  state = state.copyWith(value: value);
}
```

How to fix this error?

**Comment by AhmedLSayed9** [comment]
The conversation [here](https://github.com/rrousselGit/riverpod/issues/1879) might give you some ideas Until https://github.com/rrousselGit/riverpod/issues/1660 is done (there should be better alternatives)
